doc/src/modules/solvers/ode.rst

```diff
@@ -221,50 +221,10 @@ Linear, 2 equations, Order 1, Type 7
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order1_type7
 
-Linear, 2 equations, Order 2, Type 1
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type1
-
-Linear, 2 equations, Order 2, Type 2
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type2
-
-Linear, 2 equations, Order 2, Type 3
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type3
-
-Linear, 2 equations, Order 2, Type 4
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type4
-
-Linear, 2 equations, Order 2, Type 5
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type5
-
-Linear, 2 equations, Order 2, Type 6
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type6
-
 Linear, 2 equations, Order 2, Type 7
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. autofunction:: sympy.solvers.ode.ode::_linear_2eq_order2_type7
```

Comment:
Why is type7 still here?

Reply:
I just noticed that the type7 solver can be removed now. Earlier, I don't know why the systems of this type weren't getting solved. I would remove this solver right away, but we still need to implement a method to reduce the systems which have their minium order term greater than 0.